js/modules/k6/http/tls_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build go1.12
```

Comment:
we can just drop this IMO ;)

Reply:
Resolved by https://github.com/grafana/k6/pull/2226/commits/d87b4d6721568af0d3d6a61ec6272d0ae3f7ccbe